javascript/blogpage.js

```diff
@@ -0,0 +1,149 @@
+
+const blogitems = [
+{
+  img: 'images/blog-1.jpg',
+  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
+  h3: 'Traveling With Your Dog',
+  p: 'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat. Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+}];
+const blogitems2 = [
```

Comment:
Also sent in email, but you should combine these three into one array of blogItems.

```
const blogitems = [
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
},
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
},
{
  img: 'images/blog-1.jpg',
  alt: 'image from behind a women hugging black dog while sitting and looking at a beautiful view of a cayon',
  h3: 'Traveling With Your Dog',
  p: 'text here'
}
];
```